docs/dashboards/overview.md

```diff
@@ -124,7 +124,7 @@ From your dashboard's settings page, click "Embed Dashboard" in the left column.
 
 This will generate a code snippet, which you can drop into any HTML document at the place where you want the embedded dashboard to appear.
 
-An embedded dashboard only includes the dashboard's blocks, not its title, configuration buttons or Losant header and footer. The image below is a screenshot of a dashboard, where the sections crossed out would not appear if the dashboard were embedded.
+An embedded dashboard only includes the dashboard's blocks, not its title, configuration buttons or Losant header and footer. The image below is a screenshot of a dashboard, this is the only section that would appear if the dashboard were embedded.
```

Comment:
I'm being a little knitpicky here but I think it should read like: (basically just removes the passive tone)

The image below is a screenshot of a dashboard. This is the only content that appears when a dashboard is embedded.